superset/db_engine_specs/base.py

```diff
@@ -388,10 +389,18 @@ def df_to_sql(cls, df: pd.DataFrame, **kwargs):  # pylint: disable=invalid-name
         df.to_sql(**kwargs)
 
     @classmethod
-    def create_table_from_csv(cls, form, table):
-        """ Create table (including metadata in backend) from contents of a csv.
-        :param form: Parameters defining how to process data
-        :param table: Metadata of new table to be created
+    def create_and_fill_table_from_csv(
```

Comment:
I believe you'll also need to change the name of the `create_table_from_csv` method in the file `superset/db_engine_specs/hive.py`. The Hive `db_engine_spec` has this class in its inheritance chain and overrides this method.

Reply:
Should be fixed with [PR 31](https://github.com/Sascha-Gschwind/incubator-superset/pull/31). We now override the method in `hive.py` and changed to form-field usage so it should work.